sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/baseSender.ts

```diff
@@ -293,4 +421,56 @@ export abstract class BaseSender {
     }
     return false;
   }
+  /**
+   * Check if a metric name corresponds to a performance counter
+   * @param metricName - The name of the metric to check
+   * @returns true if the metric name is a performance counter, false otherwise
+   */
+  private isPerformanceCounterMetric(metricName: string): boolean {
+    return Object.values(BreezePerformanceCounterNames).includes(
+      metricName as BreezePerformanceCounterNames,
+    );
+  }
+
+  /**
+   * Extract telemetry type from an envelope based on its baseType
+   * @param envelope - The envelope to extract telemetry type from
+   * @returns The corresponding telemetry type
+   */
+  private getTelemetryTypeFromEnvelope(envelope: Envelope): TelemetryType {
```

Comment:
Again, this functionality is not related to the actual sender, is customer statsbeat specific, let's move the code there